admin/acceptance.py

```diff
@@ -160,7 +210,10 @@ def __init__(self):
                              % (', '.join(self.variants),))
 
     @inlineCallbacks
-    def start_nodes(self, reactor):
+    def start_nodes(self, reactor, node_count):
+        if node_count != 2:
+            raise NotImplementedError('Vagrant start_nodes must start 2 nodes')
```

Comment:
:question: I haven't looked at the callers but can this not be used to start a required one node cluster (and actually start two nodes because that's all it does)?